src/Symfony/Component/PropertyInfo/Tests/Fixtures/Php82Dummy.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\PropertyInfo\Tests\Fixtures;
```

Comment:
the license header is missing on this file

Reply:
Also added :+1: